lib/asn1/gen_glue.c

```diff
@@ -66,11 +66,11 @@ generate_int2 (const Type *t, const char *gen_name)
     Member *m;
 
     fprintf (headerfile,
-	     "%s int2%s(unsigned);\n",
+	     "%s int2%s(unsigned long);\n",
```

Comment:
Should be `unsigned long long`.

Alternatively we should use `uint64_t` and make sure we `#include <stdint.h>` anywhere we might be missing it.